kafka_utils/util/metadata.py

```diff
@@ -15,7 +15,7 @@
 from kafka import KafkaClient
 
 
-def _prepare_host_list(broker_list):
+def prepare_host_list(broker_list):
```

Comment:
I will use this one in under_replicated partitions after merging that. Now there is copy of it.